cache/in-memory/src/updates.rs

```diff
@@ -381,6 +381,16 @@ impl UpdateCache for MemberUpdate {
 
 impl UpdateCache for MessageCreate {
     fn update(&self, cache: &InMemoryCache) {
+        if cache.wants(ResourceType::USER) {
+            cache.cache_user(Cow::Borrowed(&self.author), self.guild_id);
+        }
```

Comment:
Please place newlines between logical sections of code, for example between each if statement here